maps/src/G3SkyMap.cxx

```diff
@@ -1536,7 +1536,7 @@ PYBINDINGS("maps") {
 	       "a disc of the given radius at the given sky coordinates.")
 
 	    .def("query_disc",
-	      (std::vector<size_t> (G3SkyMap::*)(quat, double) const)
+	      (std::vector<size_t> (G3SkyMap::*)(const Quat &, double) const)
```

Comment:
Is there any reason to do this? If quat is the same type as Quat basically, I'm not sure this is any more efficient.

Reply:
Fair, this made some more sense when Quat was derived from G3FrameObject.

Reply:
It looks like it's O(10ns) of overhead for the value argument vs const reference.  Pretty small, but might add up?